packages/fluentui/accessibility/src/behaviors/Accordion/accordionTitleBehavior.ts

```diff
@@ -43,7 +43,7 @@ const accordionTitleBehavior: Accessibility<AccordionTitleBehaviorProps> = props
 
 export default accordionTitleBehavior;
 
-type AccordionTitleBehaviorProps = {
+export type AccordionTitleBehaviorProps = {
   /** Element type. */
   as?: string;
```

Comment:
Please could you consider not sending `as` prop to the behavior, but instead have `isHeading` prop, which would have the logic on the component?